java/dagger/example/android/espresso/stubbed/StubNetworkingModule.java

```diff
@@ -0,0 +1,18 @@
+package dagger.example.android.espresso.stubbed;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.example.android.espresso.main.ApiClient;
+
+@Module
+public class StubNetworkingModule {
```

Comment:
That is an anti-pattern as it requires the real dependencies of that provider be available and it prevents adding test-only dependencies for the test version of the provider. Composing modules is much more powerful and is the recommended practice.